code/modules/power/lighting.dm

```diff
@@ -214,13 +214,13 @@
 	var/nightshift_enabled = FALSE	//Currently in night shift mode?
 	var/nightshift_allowed = TRUE	//Set to FALSE to never let this light get switched to night mode.
 	var/nightshift_brightness = 8
-	var/nightshift_light_power = 0.45
+	var/nightshift_light_power = 0.70
```

Comment:
This is way too close to the baseline light power (which is 0.75). With this proposed value, it'll be impossible to actually notice any difference between baseline lighting and night shift lighting, and there'd no longer be any mechanical advantages/disadvantages to night shifts (such as, but not limited to, nightmares getting some wiggle room with station lighting, podpeople needing to stay closer to station lights, botany plants growing slower with roundstart light setups, etc etc). This proposed value practically removes night shifts.